GrandCentralBoard/Widgets/Watch/Source/CalendarNameSource.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CalendarNameSource.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 12.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+class CalendarNameSource : Asynchronous {
```

Comment:
I don't like this class. Adding inheritance for such a basic configuration seems like a code smell. Never prefer inheritance over composition - it is bad in the long run and should be used with caution. I would strongly encourage you to either remove this class at all or make it configurable so that it can be used as `GoogleCalendarNameSource`.